MIGRATING.md

```diff
@@ -0,0 +1,305 @@
+# Migrate from the SignalFx Tracing Library for Go
```

Comment:
Can you update the main `READMD.md` and add 

> If you're currently using the SignalFx Tracing Library for Go and want to migrate to the Splunk Distribution of OpenTelemetry Go, see Migrate from the SignalFx Tracing Library for Go

like in Java and Python distros?